app/controllers/reviews_controller.rb

```diff
@@ -85,6 +88,28 @@ def user_params
       params.require(:review).permit(:name, :email)
     end
 
+    def delivery_params
+      params.require(:review).permit(:maxDevTime)
+    end
+
+
+    def update_maxDevTime(restaurant_id)
+      review_maxDevTime = Integer(delivery_params["maxDevTime"])
+      cur_restaurant = Restaurant.find(restaurant_id)
+      # update to the new max time
+      if cur_restaurant.maxDevTime == nil
```

Comment:
Could just be

```ruby
cur_restaurant.update!(maxDevTime: [review_maxDevTime, cur_restaurant.maxDevTime]
```

btw, I'd just call it `restaurant` and use snake_case for `max_delivery_time_minutes`